config/locales/hi.yml

```diff
@@ -0,0 +1,41 @@
+
+hi:
+  errors:
+    messages:
+      taken_in_past: पहले लिया गया 
+      equal_to_current_password: इससे अलग पासवर्ड होना चाहिए 
```

Comment:
<img width="1284" alt="Screen Shot 2020-10-08 at 10 51 19 AM" src="https://user-images.githubusercontent.com/1635820/95475425-34a40780-0954-11eb-8d7f-bb325e62fdbe.png">

This seems close but not quite right. The English version is `must be different than the current password`.

Reply:
resolved

Reply:
@salap298 thank you for making a change to this! 

I tested out adding the error to see what the full error message would look like

```ruby
self.errors.add(:password, :equal_to_current_password)
```

and it ends up as 

```
Password वर्तमान पासवर्ड के बराबर है
```

I am also going to be limited to Google Translate and miss out on the semantics, but I wanted to check. Google translates this to: 

```
Password equals current password
```

Is it possible to give more of an indication that the password should be **different** than the current password? For example, our English translation would be

```
Password must be different than the current password
```

 If not, I think we would be okay - if you get an error when updating your password that it's the same as your current that _should_ be a sign that they shouldn't match. However, if possible, I think it could be useful to be even more explicit.